scripts/main.js

```diff
@@ -31,6 +32,65 @@ module.exports = (robot) => {
             }
         }
     })
+    
+    robot.hear(/bot create milestone|bot close milestone/i, (res) => {
+        var message = res.message.text
+        message = message.split(' ')
+        if(message.length === 7 && message[0] === "bot" && message[1] === "create" && message[2] === "milestone") {
```

Comment:
Why should the length of the message be exactly 7?

Reply:
@zelima It is all about validation, otherwise it will run a command if it finds this structure in the middle of a sentence.